test/vlq-tests.ts

```diff
@@ -109,29 +109,27 @@ describe('test encode', function() {
   it('encoder', function() {
     // (lines + segemnts * 6) = byte_count
     let decoded = {
-      mappings: {
```

Comment:
> This PR is already not very focused

Sorry about that. If you want I can split it up a bit more. The commits themselves should be reasonably self-contained so you could also review them one by one if that helps with the focus.